framework/source/class/qx/event/handler/Focus.js

```diff
@@ -1063,8 +1063,9 @@ qx.Class.define("qx.event.handler.Focus",
           return focusedElement;
         }
         // Check compound widgets
-        var widget = qx.ui.core.Widget.getWidgetByElement(focusedElement),
-          textField = widget && widget.getChildControl && widget.getChildControl("textfield", true);
+        var Widget = qx.Class.getByName("qx.ui.core.Widget");
```

Comment:
I think Widget is a somewhat 'reserved' word ... https://www.w3.org/TR/widgets-apis/#widget ... also using that same word in upper and lowercase spelling does not feel right ...

Reply:
The `Widget` (capital W) is just a way to get the `qx.ui.core.Widget` class without triggering a compiler dependency, so as a class name it's correct

Reply:
your intention is clear to me but at least in github the word `Widget` with capital W gets syntax highlighting as if it had special meaning ... that might be confusing to people looking at the code ... maybe call it `WidgetClass` instead

Reply:
I agree with @johnspackman. This is a qooxdoo convention, that a shorthand for a class is the final component of the class name with a leading capital letter as in that final component. The fact that it gets highlighted in some editor isn't, IMO, sufficient reason to break our standard convention.